resources/views/pages/partials/email-subscription.blade.php

```diff
@@ -1,7 +1,7 @@
 <div class="form-group">
     <h1>Email subscription</h1>
     <p class="lead">
-      Creates/updates users from uploaded CSV, expecting an <code>email</code> column header.
+      Creates/updates users from uploaded CSV, expecting <code>email</code> and <code>first_name</code> column headers.
```

Comment:
Can we remove the [`display: block` styling for `code` tags](https://github.com/DoSomething/chompy/blob/104977bd33e6a6022177d24f2af17d16f30437e3/resources/assets/sass/app.scss#L31-L33) (and if necessary, apply it with a more specific selector wherever it was necessary)? It makes this kinda awkward to read.

Reply:
I'm not sure where we use this style override, but it might also make more sense to use a [`pre`](https://getbootstrap.com/docs/3.4/css/#code-block) tag there instead, rather than overloading [the inline code element](https://getbootstrap.com/docs/3.4/css/#code-inline) across the board.

Reply:
> I'm not sure where we use this style override, but it might also make more sense to use a pre tag there instead, rather than overloading the inline code element across the board.

@DFurnes ah interesting! For this though, I think we only want one word at a time wrapped in the `<code>` tag. `<pre>` looks like it is used when you want multiple lines selected, unless I'm understanding incorrectly?

Reply:
That's correct!

The elements on this page should be `code` tags _without_ the `display: block` override. If we do want this `display: block` override somewhere else, we should use a `pre` tag for that specific case.

Reply:
ohoh I see what you're saying now! Thanks @DFurnes ! I think this is the only place other than this file that has it and with the `display: block` taken out, looks better to me too!
![Screen Shot 2019-06-18 at 2 30 07 PM](https://user-images.githubusercontent.com/9019452/59709841-b018fd00-91d5-11e9-867a-95be925cce01.png)

Reply:
hmm I removed the code from `app.css` but for some reason it isn't registering the changes when I try to add to GitHub?

Reply:
oh nvm, got it!